docs/application/tizen-studio/native-tools/batt-historian.md

```diff
@@ -0,0 +1,157 @@
+# Battery Historian
+
+This page explains how you can use the Battery Historian tool to analyze the battery-consumption patterns, and also helps you to understand, interpret, and evaluate the reports generated by the Battery Historian tool.
+
+Battery Historian tool helps you to increase the productivity of Tizen devices by maximizing the performance with efficient use of resources. Battery Historian tool enables you to analyze and audit the behavior and consumption of battery in the Tizen devices. It is an expert analysis tool that precisely reviews information and events related to the battery. It includes various metrics and insights, such as radio cells, communication, and so on. Battery Historian tool generates a report that helps you to diagnose and troubleshoot the device behavior related to battery consumption. The generated report also helps you to identify precise battery-draining elements so that you can fine-tune and augment Tizen device productivity and efficiency.
```

Comment:
"maximizing the performance" => maximize power efficiency? Performance is very generic, which is applicable to many other factors not just power.

Reply:
@sijupunnnoose thanks! 

The introduction starts with generic sentences, then the introduction narrows down to specifics of the Battery Historian tool.  

Battery Historian tool helps you to increase the **productivity** of Tizen devices by maximizing the **performance** with **efficient use of resources**.

**Productivity:**   What I intended to convey to the readers. 

It is the how efficiently the device performs the tasks it is intended to do. So, in this case, if we analyze and diagnose why the battery is draining more power and troubleshoot it, we can increase the overall productivity of any battery-based Tizen device as the battery will last more hence the device will be more productive. 

So, if resources are used judiciously and analyzed properly,  there will be sure gain in performance (margin may be less) in general for any device.

If you think otherwise, let me know I will rephrase is as per your thoughts.